gitbutler-ui/src/lib/components/KeysForm.svelte

```diff
@@ -9,6 +9,7 @@
 	import Link from '$lib/components/Link.svelte';
 	import { copyToClipboard } from '$lib/utils/clipboard';
 	import { debounce } from '$lib/utils/debounce';
+	import { open } from '@tauri-apps/api/shell';
```

Comment:
Can we use `openExternalUrl` here instead? This has happened before because `open` is a global type.

Reply:
this make sense @mtsgrd. I should have checked the definition for `openExternalUrl` earlier, my bad.
Addressed your review comment with [this commit](https://github.com/gitbutlerapp/gitbutler/pull/2984/commits/01cff8eaac5e48bd1aacb56765387cf23b4b0db8).